components/SecondHeading/SecondHeadingNonSticky.styled.js

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+const SecondHeadingNonSticky = styled.h3`
+  text-align: left;
+  margin: 0.5rem;
+  padding: 0.5rem;
+  top: 0;
+  width: 100%;
+  background-color: var(--white);
+`;
+export default SecondHeadingNonSticky;
```

Comment:
Instead of having two different styled components (`SecondHeadingSticky` and `SecondHeadingNonSticky`) you should consider using props ([Adapting based on props](https://styled-components.com/docs/basics#adapting-based-on-props)). You could have one component `SecondHeading` with a boolean prop `sticky` for example.

Reply:
Hey @actyralf thank you for the feedback! I tried to summarize fitting styles so I guess now it's more clearly arranged now ☺️